sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala

```diff
@@ -823,4 +823,21 @@ class JsonFunctionsSuite extends QueryTest with SharedSparkSession {
       }
     }
   }
+
+  test("SPARK-32968: json pruning optimization with corrupt record field") {
```

Comment:
Thanks, @viirya . Since this PR is a follow-up for SPARK-33907, shall we use `SPARK-33907`?

Or, `SPARK-32968` is still not merged, shall we wait for SPARK-32968?

Reply:
Oops, this is a mistake.